packages/core/js-client/src/compilerSupport/callFunction.ts

```diff
@@ -40,13 +41,13 @@ const log = logger('aqua');
  * @param args - args in the form of JSON where each key corresponds to the name of the argument
  * @returns
  */
-export const callAquaFunction: CallAquaFunctionType = ({ def, script, config, peer, args }) => {
+export const callAquaFunction: CallAquaFunctionType = async ({ def, script, config, peer, args }) => {
```

Comment:
That's a big change in the API.

Who depends on `callAquaFunction`? Gotta check they are aware of this change, and prepared for it.

Reply:
Nothing is changed actually. `callAquaFunction` returned promise before the changes.

Reply:
ah, missed that. good! :)